rust/platform/triple_mappings.bzl

```
@@ -212,7 +212,10 @@ def triple_to_system(triple):
     if len(component_parts) < 3:
         fail("Expected target triple to contain at least three sections separated by '-'")
 
-    return component_parts[2]
+    system = component_parts[2]
+    if (system == "androideabi"):
```

Small nit for consistency's sake, if you don't mind
```suggestion
    if system == "androideabi":
```

---

Done is a child commit.